common/src/main/java/com/wynntils/models/lootrun/LootrunModel.java

```diff
@@ -246,6 +266,9 @@ public void onCharacterChange(CharacterUpdateEvent event) {
         selectedBeacons = selectedBeaconsStorage.get().get(id);
 
         selectedBeaconsStorage.touched();
+
+        missionStorage.get().putIfAbsent(id, new LinkedList<>());
```

Comment:
```suggestion
        missionStorage.get().putIfAbsent(id, new ArrayList<>());
```

Storages use `ArrayList` as the implementation for lists. Do you need LinkedLists specifically?

Reply:
The order of the missions needs to be retained in the storage. Since the values are already being stored in strings, I could  append prefixes to them to order them once fetched from storage, though it would be a rather messy solution.

Reply:
ArrayLists already retain the order? We are not talking about sets here

Reply:
My apologies, I mixed up List index order with Map key index order.